src/globals.cpp

```diff
@@ -265,6 +268,8 @@ CTweak<uint64_t> pruneIntervalTweak("prune.pruneInterval",
 
 CTweak<uint32_t> netMagic("net.magic", "network prefix override. If 0 use the default", 0);
 
+CTweak<uint32_t> randomlyDontInv("net.randomlyDontInv", "Skip sending an INV for some percent of transactions", 0);
```

Comment:
should we qualify this with "for every node that supports mempool sync"?

Reply:
The way it is implemented now, any node can use the switch (although it is perhaps not a good idea if mempool sync is disabled). I can, however, make it available only to nodes that have enabled mempool sync and adjust the message accordingly.